app/controllers/admin/worldwide_offices_controller.rb

```diff
@@ -23,7 +23,7 @@ def update
 
     if (contact_numbers_attributes = update_params.dig(:contact_attributes, :contact_numbers_attributes))
       contact_numbers_attributes.each do |_id, contact_number_attribute|
-        contact_number_attribute[:_destroy] = true if contact_number_attribute[:label].empty? && contact_number_attribute[:number].empty?
+        contact_number_attribute[:_destroy] = true unless contact_number_attribute[:label].present? && contact_number_attribute[:number].present?
```

Comment:
Should the `&&` be `||` here now? Or is the contact number only valid if **both** fields are present?

Reply:
Only if both are present that's correct